backend/entities/ParcelFiscal.cs

```diff
@@ -26,6 +26,12 @@ public class ParcelFiscal : BaseEntity
         /// <value></value>
         public int FiscalYear { get; set; }
 
+        /// <summary>
+        /// get/set - The effective date of this fiscal value
+        /// </summary>
+        /// <value></value>
+        public DateTime? EffectiveDate { get; set; }
```

Comment:
`BuildingFiscal` is not nullable, but here a `ParselFiscal` is?  Which is correct?

Reply:
they should both be nullable.